_includes/specials/bech32/16-checklist.md

```diff
@@ -0,0 +1,54 @@
+Recently, an Optech contributor surveyed many popular wallets and Bitcoin
+exchanges to see what technical features they supported.  For one of the
+exchanges, he initially recorded it as supporting sending to bech32
+addresses---but later he discovered its support wasn't entirely
+complete.
+
+The problem was that the exchange supported P2WPKH bech32 addresses
+(single-sig addresses) but not P2WSH bech32 addresses (multisig and
+complex script addresses).  Another problem was accepting all-lowercase
+bech32 addresses but not all-uppercase bech32 addresses.  A different
+exchange limited the length of address form fields so that they couldn't
+fit all valid bech32 addresses.
+
+With these problems in mind, we've created a short checklist for testing
+basic bech32 sending support:
+
+1. Generate two addresses of your own, one for P2WPKH and one for P2WSH.
+   For example, using Bitcoin Core, the [jq][] JSON parser, and the Bash
+   shell, you can run the following commands:
+
+     ```bash
+     $ p2wpkh=$( bitcoin-cli getnewaddress "bech32 test" bech32 )
+     $ p2wsh=$(
+       bitcoin-cli addmultisigaddress 1 \[$(
+         bitcoin-cli getaddressinfo $p2wpkh | jq .pubkey
+       )\] "bech32 test" bech32 | jq -r .address
+     )
+     $ echo $p2wpkh $p2wsh
+     $ echo $p2wpkh $p2wsh | tr '[a-z]' '[A-Z]'
+     ```
+
+2. Test sending a small amount of bitcoin to each lowercase address
```

Comment:
Should we add some additional warnings here:

- backup the wallet used to generate the addresses to make sure the funds don't get lost
- do this on testnet before trying on mainnet
- A final warning like "WARNING! If your software does not correctly implement sending to bech32 addresses these funds may become unspendable. Only send a minimal amount that you can afford to lose."

(may be overkill, but always better to be safe I think)

Reply:
I'm adding a line about "only using a small amount you can afford to lose."  I think the rest of these suggestions are unnecessary if that rule is followed.